data/xsl/standings.css

```diff
@@ -0,0 +1,117 @@
+// $Id$
```

Comment:
Isn't the above line a reference to an obsolete source control system?  Does it actually get used somewhere?